src/DurableTask.Netherite/StorageLayer/Faster/StoreWorker.cs

```diff
@@ -281,6 +282,42 @@ internal enum CheckpointTrigger
             Idle
         }
 
+        void LogCheckpointStats()
+        {
+            long inputQueuePositionLag = this.getInputQueuePositionLag();
+            this.CheckpointDue(out CheckpointTrigger trigger, out long? compactUntil);
+            var reportNullableTaskStatus = (Task? t) =>
```

Comment:
For improved performance, consider using a static local function instead of an `Action<Task>`. Something like:
```suggestion
            static string ReportNullableTaskStatus(Task? t)
```

Reply:
Clever, didn't know we could create static _yet local_ methods. Incorporated in my next commit.